src/time_tracker/invoices/handlers/spec.clj

```diff
@@ -45,6 +45,5 @@
 (s/def ::timers (timers-spec/normalized-timers-spec 0))
 
 (s/def ::invoice-data
-  (s/and (s/merge ::generate-invoice-params-coerced
-                  (s/keys :req-un [::users-spec/users ::timers-spec/timers]))
-         invoice-data-pred))
```

Comment:
@trycatcher rather than deleting this predicate, please modify it to check that for each user in `users`, there is a corresponding `timer`, and vice versa. That is, the set of user IDs in `users` should be the same as the set of user IDs in `timers`.